README.md

```diff
@@ -0,0 +1,6 @@
+## README.md
+Wmjanderson 3/28/17
+
+This repo is for the rest of my stuff in teamtreehouse.com C# track.
+
+*Salud*, your bud #iWill#
```

Comment:
iWill is the pseudonym for ##willanderson##, the crackpot who writ this garbage.